fdbserver/workloads/RemoveServersSafely.actor.cpp

```diff
@@ -693,10 +765,188 @@ struct RemoveServersSafelyWorkload : TestWorkload {
 		return subset;
 	}
 
+	template <class T>
+	static std::set<T> random_subset(std::unordered_set<T> s, int n) {
+		std::vector<T> v;
```

Comment:
The `std::copy` is unnecessary as `std::vector` already has a constructor that takes two iterators. I would write this as the one-liner:

```
return random_subset(std::vector<T>(s.begin(), s.end()), n);
```